CMakeLists.txt

```diff
@@ -1897,3 +1897,14 @@ if(LLVM_TOOLCHAIN_CROSS_BUILD_MINGW)
         mingw-llvm
     )
 endif()
+
+add_custom_target(check-all-embedded-toolchain)
+
+add_dependencies(
+    check-all-embedded-toolchain
+    check-all
+    check-llvm-toolchain
+    check-package-llvm-toolchain
+    check-cxxabi
```

Comment:
Is `check-cxx` missing? I can see `check-cxxabi` and `check-unwind` 

I would have intuitively expected `check-all` to include all the LLVM components including runtimes but I guess this isn't the case.